gmsh_utils/gmsh_IO.py

```diff
@@ -181,7 +185,9 @@ def create_volume_by_extruding_surface(self, surface_id: int,
                                               extrusion_length[2])
         # gets the first volume tag from the list of new dimension tags
         volume_tag: int = next((dim_tag[1] for dim_tag in new_dim_tags if dim_tag[0] == volume_dim))
-        gmsh.model.setPhysicalName(volume_dim, volume_tag, name_label)
+        if name_label != "":
+            gmsh.model.setPhysicalName(volume_dim, volume_tag, name_label)
```

Comment:
can you check if "setPhysicalName" is required?, since you also add the name in "addPhysicalGroup"